parachain/pallets/dot-app/src/lib.rs

```diff
@@ -71,6 +71,16 @@ decl_storage! {
 		/// Address of the peer application on the Ethereum side.
 		Address get(fn address) config(): H160;
 	}
+
+	add_extra_genesis {
+		config(initial_module_balance): BalanceOf<T>;
```

Comment:
See my other comment about whether its possible to initialize the locked balance in `dispatch_benchmarks`.

If that doesn't work, the current approach works I guess. Though can we rename `initial_module_balance` to `balance`? Its simpler.